src/git/commit.js

```diff
@@ -18,7 +18,7 @@ function commit(sh, repoPath, message, options, done) {
   // On windows we must use an array in gulp-git instead of a string because
   // command line parsing works differently
   if(os.platform()=="win32") {
-    operatingSystemNormalizedMessage = escapedMessage.split(/\r?\n/);
+    operatingSystemNormalizedMessage = escapedMessage.replace(/\r/g, '');
```

Comment:
Did something in gulp-git change so that it now works with a string? Or is the above comment incorrect?